### PR TITLE
Increase payload limit to 100KB

### PR DESCRIPTION
**Pull request by ruanxin**

### CODE REVIEW

File: cmd/mothership/mothership/start/http.go [hunk 1]
```diff
@@ -127,11 +127,6 @@ func startWebserver(ctx context.Context, o *Options) error {
 		callHandler(o, createOrUpdateCluster)).
 		Methods(http.MethodPost, http.MethodPut)
 
-	apiRouter.HandleFunc(
```
Comment (jakobmoellersap) [inline cmd/mothership/mothership/start/http.go:130]: This for sure is not intended right?

**Review by jakobmoellersap — CHANGES_REQUESTED** [review]

File: cmd/mothership/mothership/start/http.go (hunk 1 above)
Comment (ruanxin) [reply cmd/mothership/mothership/start/http.go:130]: actually intended, if you check the block above, it's duplicated. I'm really questioning the review process of other team...

**Review by jakobmoellersap — APPROVED** [review]

**Pull request merged**